logstreams/src/main/java/io/camunda/zeebe/logstreams/impl/log/LogStreamBatchWriterImpl.java

```diff
@@ -263,6 +277,9 @@ private void writeEventsToBuffer(
       final int bufferOffset = claimedBatch.getFragmentOffset();
 
       // write log entry header
+      if (skipProcessing != 0) {
+        LogEntryDescriptor.skipProcessing(writeBuffer, bufferOffset);
+      }
```

Comment:
:wrench: Ok I see here that it is not that nice to check and then set :D I guess directly being able to set the byte would be nice, but this would change other parts I guess. Plus it is not really clear what the byte means (I guess except we have  a constant).

Reply:
I introduced constants